src/features/world/mmoMachine.ts

```diff
@@ -32,18 +32,46 @@ export type ServerId =
   | "sunflorea_brazil"
   | "sunflorea_magic";
 
-type ServerName = "Bliss" | "Dream" | "Oasis" | "Brazil" | "Magic";
+type ServerName =
+  | "Bliss"
+  | "Dream"
+  | "Oasis"
+  | "Brazil"
+  | "Magic"
+  | "Bumpkin Bazaar";
+type ServerPurposes = "Chill & Chat" | "Trading";
```

Comment:
Small thing, the type should be singular ie `ServerPurpose` as it will only ever be one.

Reply:
Ooops, nice find thank you ❤️